tests/integration/test_installation.py

```diff
@@ -100,6 +108,39 @@ def test_job_failure_propagates_correct_error_message_and_logs(ws, sql_backend,
     assert len(workflow_run_logs) == 1
 
 
+def test_job_cluster_policy(ws, new_installation):
+    install = new_installation(lambda wc: replace(wc, override_clusters=None))
+    cluster_policy = ws.cluster_policies.get(policy_id=install.config.policy_id)
+    policy_definition = json.loads(cluster_policy.definition)
+
+    assert cluster_policy.name == f"Unity Catalog Migration ({install.config.inventory_database})"
+
+    assert policy_definition["spark_version"]["value"] == "14.3.x-scala2.12"
+    assert policy_definition["node_type_id"]["value"] == "Standard_F4s"
+    assert policy_definition["azure_attributes.availability"]["value"] == "ON_DEMAND_AZURE"
+
+
+@pytest.mark.skip
```

Comment:
Why skip? Why new test?

Reply:
I want to run the test using new cluster installation to check if all policies are applied correctly and I ran the assessment test. But we don't need to run this for int test so I tested it manually and marked as skipped